api/api.proto

```diff
@@ -1190,6 +1190,54 @@ service OpenStorageClusterPair{
     }
 }
 
+// OpenStorageClusterDomains is a service used to manage cluster domains in an openstorage cluster.
+// A single openstorage cluster can stretch across multiple cluster domains. In other words it can
+// span across multiple Container Orchestrator clusters. Each node in the cluster is assigned a
+// a cluster domain to start with. A cluster domain and in turn the nodes which are a part of it
+// are assigned a status indicating whether the cluster domain is actively participating in the
+// cluster or whether it is inactive.
+service OpenStorageClusterDomains {
+  // Enumerate returns names of all the cluster domains in the cluster
+  rpc Enumerate(SdkClusterDomainsEnumerateRequest)
+    returns (SdkClusterDomainsEnumerateResponse) {
+      option(google.api.http) = {
+        get: "/v1/cluster/clusterdomains"
```

Comment:
all rest calls should fall under their service, so these should be `/v1/clusterdomains/...`